### DISCUSSION TITLE
Different network segments cannot display each other's audio and video

### DISCUSSION THREAD

**Discussion by TroyeYing** [pion/webrtc#1953] [Q&A] [answered]
may I ask why webrtc calls successfully in different network segments, but can't display the other party's audio and video?

**Comment by Sean-Der** [comment, marked as answer]
Hi @TroyeYing 

Two PeerConnections will be able to signal, but will not establish ICE Connectivity. You can check `iceconnectionstate` in any WebRTC implementation to confirm you have connectivity. To establish connectivity you probably will need to run a STUN/TURN server.

To learn more about WebRTC Connectivity [WebRTC for the Curious#Connecting](https://webrtcforthecurious.com/docs/03-connecting/)
